hivemind/p2p/servicer.py

```diff
@@ -131,6 +131,7 @@ async def add_p2p_handlers(self, p2p: P2P, wrapper: Any = None, *, namespace: Op
                 getattr(servicer, handler.method_name),
                 handler.request_type,
                 stream_input=handler.stream_input,
```

Comment:
Nitpicking: should we create them asynchronously via gather

Reply:
upd: this is definitely not a bottleneck, feel free to ignore/resolve

Reply:
I support this because it may improve startup time. Good catch!

Reply:
Done!